sentry-android-ndk/src/main/java/io/sentry/android/ndk/SentryNdk.java

```diff
@@ -5,5 +5,14 @@
     System.loadLibrary("sentry");
   }
 
-  public static void init() {}
+  static {
+    System.loadLibrary("sentry-android");
```

Comment:
You may want to consider usage of [ReLinker](https://github.com/KeepSafe/ReLinker) (suggestion [here](https://developer.android.com/training/articles/perf-jni#native-libraries))

Reply:
That does sound like a good suggestion. Although I'm not sure it's worth the extra dependency. /cc @mitsuhiko

Reply:
that makes sense, even the official documentation assumes bugs on older versions and tag Relinker as a solver, I'd go for it, ReLinker also has Asynchronous loading which could faster App. init.

Reply:
Let's do this on a separate PR to avoid keeping this open for longer

Reply:
I've opened an issue internally to track this